### DISCUSSION TITLE
Is Lolin32 lite board supported?

### DISCUSSION THREAD

**Discussion by Vicente-M-D** [pschatzmann/ESP32-A2DP#520] [Q&A] [answered]
Hi there, 
According to the manufacture this little board (lolin32 lite) includes an ESP32-DOWDQ6, is this microcontroller supported? I am trying my best to make work this board together with the external DAC PCM5102. The DAC was successfully tested with the same code and pins assignation in a DOIT Esp32 DevKit v1. 
The Lolin32 does not provide 5V pin so, the DAC has to be supplied by 3,3V which I thought it was the problem. But, I connected the DAC to 5V by an external power supply, the Lolin32 lite to usb connector and all grounds together and...nothing.  

The lolin32 has a connector for external battery which is ideal for my purpose, a portable Bluetooth speaker however, I starting to think that this board will never work with the library BluetoothA2DPSink.

Thanks in advance and congratulations for your work and support.

**Comment by elektryk2** [comment, marked as answer]
> I starting to think that this board will never work with the library BluetoothA2DPSink.

I've got this board connected to ESS9023 DAC and it works ok with BluetoothA2DPSink.

**Reply by pschatzmann** [reply]
I just noticed that I don't get any e-mail notifications for the discussions any more...
